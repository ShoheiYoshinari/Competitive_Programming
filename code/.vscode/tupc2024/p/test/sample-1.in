4
4
2 6 7 3
2
1 1000000000
20
1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
8
123 4567 8912 34 56789 12345 6789 1
