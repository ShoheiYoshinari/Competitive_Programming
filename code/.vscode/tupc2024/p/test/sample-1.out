12
1000000000
10
72647
